eis_toolkit/feature_importance/feature_importance.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+import pandas
+import pandas as pd
+import sklearn.neural_network
+from sklearn.inspection import permutation_importance
+
+from eis_toolkit.exceptions import InvalidDatasetException
+
+
+def evaluate_feature_importance(
+    clf: sklearn.neural_network or sklearn.linear_model,
+    x_test: np.ndarray,
+    y_test: np.ndarray,
+    feature_names: list[str],
+    number_of_repetition: int = 50,
+    random_state: int = 0,
+) -> (pandas.DataFrame, dict):
+    """
+    Evaluate the feature importance of a sklearn classifier or linear model.
+
+    Parameters:
+        clf (Any sklearn nn model or lm model): Trained classifier.
```

Comment:
Regarding the docstring, no need to include the type in paranthesis (just a decision we made).

Reply:
I removed type than!